tools/nuget/README.md

```diff
@@ -0,0 +1,44 @@
+# NuGet package creation process
+
+This directory contains NuGet specification files (.nuspec).
+
+Creating nuget packages:
+
+1. Set `PackageVersion` environment variable that defines the package version.
+
+```console
+set PackageVersion=0.2.0
+```
+
+If environment variable is not set, then the version is automatically obtained from git tag:
+
+```console
+git describe --tags
+```
+
+2. Run `tools/build-nuget.cmd` script to create the package(s). Packages are copied to `.\packages` directory.
+
+3. Push selected package to nuget feed.
+
+Learn more about NuGet packages [here](https://docs.microsoft.com/en-us/nuget/guides/native-packages).
+
+## Proposed NuGet package flavors
+
+This section is a `DRAFT` - Work-in-Progress.
+
+The following packages MAY be created and published at nuget.org :
+
+| Package Name                 | Desription                                                  | Notes |
+|------------------------------|-------------------------------------------------------------|-------|
+| OpenTelemetry.Cpp.Source     | Complete recursive source code snapshot with dependencies.  |       |
+| OpenTelemetry.Cpp.Dev        | Source code snapshot excluding 3rd party dependencies.      |       |
+| OpenTelemetry.Cpp.Api        | API headers package.                                        |       |
+| OpenTelemetry.Cpp.Sdk        | SDK headers and source package.                             |       |
+| OpenTelemetry.Cpp.Ext        | Extended functionality.                                     |       |
```

Comment:
Is this API extension or SDK extension or both?

Reply:
It's the contents of `ext` directory:
- zpages
- http client curl
- http server

These are optional components and should not be generally required for building SDK.